### ISSUE TITLE
Workflow & Initial Setup

### ISSUE THREAD

**Issue by atuttle** [atuttle/PhoneGapTips#1]
Here's what I'm thinking:

- Static site generator (TBD), which avoids yet-another-password syndrome, reduces the ability to have the website compromised, etc
- Hosted either here on gh-pages, or on surge.sh
- Each issue we'll collaborate on a file (type TBD by site generator?) that will then be used to generate the archives _as well as the email content_.
- Community links/tips/etc can be submitted as issues.

I think that's it for now. Other ideas?

**Comment by cdibened** [comment]
Sounds great.  As long as others can submit posts and/or we can include posts(permitted of course) from other well-known devs, such as @cfjedimaster, will only add to the credibility and enhance the content of the site at the same time.

**Comment by cfjedimaster** [comment]
So we're posting the tips on the site too? I thought it was just a newsletter. Don't get me wrong - I totally approve.

1) Jekyll or Harp. (Jekyll is more powerful but Harp is simpler.) Either way - I'd be happy to volunteer to handle that part.

2) Surge.sh is better imo.

**Comment by cdibened** [comment]
@cfjedimaster Sorry for the confusion.  What I really meant was having links with an abstract to posts on your site.

**Comment by cfjedimaster** [comment]
I wasn't confused by you, but by @atuttle :)

**Comment by cdibened** [comment]
@cfjedimaster Maybe I added to the confusion by saying "site" instead of newsletter.

**Comment by cfjedimaster** [comment]
Heh well I think @atuttle started it. ;) As I said, I approve of the idea.

**Comment by cdibened** [comment]
And +1 for surge.sh

**Comment by coldfumonkeh** [comment]
+1 for surge

**Comment by atuttle** [comment]
Let me clear up some of the confusion.

When I thought I was going to be doing this by myself, I figured I'd send out some original content of my own mixed in with the links -- stuff you could only get by signing up for the newsletter. With a bunch of us contributing, I'm not sure that makes as much sense. ...Unless we tack on a blog in the site (which we'd use a category for newsletter archives, rather than the whole thing being newsletter archives), and then we can contribute our original content on that blog, and link to it from the newsletter.

While I **don't** think we want to _just_ emulate the Cooper Press "XYZ Weekly" format of:

> [**Some interesting link title**](http://foo.bar)
> Some custom blurb we write about the article

I **do** want this mostly to be aggregation of interesting and useful content from elsewhere. (There's tons of great content out there, so let's not be arrogant and say only ours is worth reading.) So we'll have _some_ content like that, but also some longer form sections. Closer to the [Sublime Text Tips](http://sublimetexttips.com/) format. (Where relevant, we may create & embed screen shots or even animated gifs to show off what's being discussed, for example...)

---

The static-site bit was a thought I had about how we can generate the archives and the email content from the same source material... and being static has lots of other benefits too (easy hosting, hack-proof, etc).

Hopefully that helps?

**Comment by cdibened** [comment]
Got it. Clearer picture.

**Comment by atuttle** [comment]
I got us linked in the next issue of Mobile Web Weekly. Hopefully that'll provide a nice bump in subscribers. Currently we're at a hair over 70. I figure once we get a couple hundred subscribers, if we haven't started preparing the first issue yet, we should jump on it.

Personally I'm basically underwater right now. I'm working ridiculous hours from now at least until early September. I'm ok letting this ride with just the subscribe form for that long, but I'd like to have the first issue out by mid-to-late September at the latest.

**Comment by cdibened** [comment]
Oh man, that's cool.  Like I mentioned to you in slack, I'm releasing an app today so for the next week most of my effort will be on that but I'll start writing up a high-level list of topics that can be discussed for the first issue.

**Comment by atuttle** [comment]
I created #2 for collecting _and discussing_ source material for issue 1. :)

**Comment by atuttle** [comment]
Another interesting free* static host: https://www.netlify.com/

They have github integration too, which means deploying updates could be as easy as git push / merging pull requests.

* They have a branded "callout" which I'm not sure what it looks like... will need to investigate.

**Comment by atuttle** [comment]
I put up a static site on Surge in the last few days and I really like it so far. Very easy to use, and collaboration is easy too.

**Comment by atuttle** [comment]
> I got us linked in the next issue of Mobile Web Weekly. Hopefully that'll provide a nice bump in subscribers. Currently we're at a hair over 70. I figure once we get a couple hundred subscribers, if we haven't started preparing the first issue yet, we should jump on it.

That issue went out, and it was a significant bump! As of right now we're sitting at 190 subscribers, and they continue to trickle in.

There's also [the official PhoneGap newsletter](http://phonegap.com/community/newsletter/) which could get us in front of a lot of people, too. I believe there are multiple people that can get us mentioned in that, and I'm working on it.

Right now the focus is on driving subscribers. Any other ideas?

**Comment by coldfumonkeh** [comment]
I can hit up Colene and Brian to help get into the official newsletter.
Awesome work with the subs.

**Comment by cfjedimaster** [comment]
Matt, if you don't hear back let me know and I'll ask Holly.

On Fri, Aug 21, 2015 at 9:42 AM, Matt Gifford <notifications@github.com>
wrote:

> I can hit up Colene and Brian to help get into the official newsletter.
> Awesome work with the subs.
>
> On Fri, Aug 21, 2015 at 3:39 PM, Adam Tuttle <notifications@github.com>
> wrote:
>
> > I got us linked in the next issue of Mobile Web Weekly. Hopefully that'll
> > provide a nice bump in subscribers. Currently we're at a hair over 70. I
> > figure once we get a couple hundred subscribers, if we haven't started
> > preparing the first issue yet, we should jump on it.
> >
> > That issue went out, and it was a significant bump! As of right now we're
> > sitting at 190 subscribers, and they continue to trickle in.
> >
> > There's also the official PhoneGap newsletter
> > <http://phonegap.com/community/newsletter/> which could get us in front
> > of a lot of people, too. I believe there are multiple people that can get
> > us mentioned in that, and I'm working on it.
> >
> > Right now the focus is on driving subscribers. Any other ideas?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/atuttle/PhoneGapTips/issues/1#issuecomment-133447891
> >.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atuttle/PhoneGapTips/issues/1#issuecomment-133448536>.
>



-- 
===========================================================================
Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden

**Comment by atuttle** [comment]
Brian Rinaldi is introducing me to Holly over email today*, so hopefully I'll have that done soon.

* I think

**Comment by atuttle** [comment]
You all still good to go? I'd like to get issue 1 out at the end of this week if possible. We don't have to have the website up or anything other than the content, really. Just time to gather links and anything else interesting and post it in #2

**Comment by cfjedimaster** [comment]
Adam, forgive me, but the last month has been hellish on me due to travel.
Can you remind me what you need from me initially?

On Mon, Sep 21, 2015 at 7:21 AM, Adam Tuttle <notifications@github.com>
wrote:

> You all still good to go? I'd like to get issue 1 out at the end of this
> week if possible. We don't have to have the website up or anything other
> than the content, really. Just time to gather links and anything else
> interesting and post it in #2
> <https://github.com/atuttle/PhoneGapTips/issues/2>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atuttle/PhoneGapTips/issues/1#issuecomment-141955258>.
>



-- 
===========================================================================
Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden

**Comment by atuttle** [comment]
I think all that we need in the short term is to get our first few issues out. Let's shoot for every other week to start, and if we have enough content we can increase that to every week.

The website can wait. I'll figure out an email template. So we just need to figure out what content we want to include. Maybe start taking the things from #2 (and more!) and working up a markdown file in this repo to be the body of the message.

I'd also like to think of this in sort of a magazine or newspaper style, too (in some ways). Like for example, what about a recurring "column" where someone would write about performance, or newly available plugins, or just compare/contrast plugins that do the same thing, etc.

**Comment by cfjedimaster** [comment]
I see we have 9 links (#2) - I say ship it. Unless you want one more... I'm adding one now.

**Comment by atuttle** [comment]
Yep, I just put the email together. It'll go out on Monday morning. Let's hype the subscribe form with that fact! Here's a tweet you can share: https://twitter.com/PhoneGapTips/status/649964873137393664

**Comment by cfjedimaster** [comment]
Done!

On Fri, Oct 2, 2015 at 10:20 AM, Adam Tuttle <notifications@github.com>
wrote:

> Yep, I just put the email together. It'll go out on Monday morning. Let's
> hype the subscribe form with that fact! Here's a tweet you can share:
> https://twitter.com/PhoneGapTips/status/649964873137393664
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atuttle/PhoneGapTips/issues/1#issuecomment-145059571>.
>



-- 
===========================================================================
Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden
